.travis.yml

```diff
@@ -26,7 +26,34 @@ after_script:
   - if [[ $TRAVIS_PHP_VERSION = '7.0' && $DB = 'sqlite' ]]; then php ocular.phar code-coverage:upload --format=php-clover build/logs/clover.xml; fi
 
 matrix:
+  fast_finish: true
   include:
+    - php: hhvm-3.15
```

Comment:
This is painful to maintain, and will require changing every time there is a minor version bump. Can we just rely on `hhvm` at the base config, and have travis-ci update to latest stable?